pkg/eventcollector/eventcollector.go

```diff
@@ -153,11 +153,47 @@ func (c *EventCollector) handleEvent(payload interface{}) interface{} {
 	err = c.callTriggers(event)
 	if err != nil {
 		log.Errorf("Error calling triggers: err: %v", err)
+		errors <- err
```

Comment:
The error from callTriggers wasn't being sent to the errors channel so I added this. Is this wrong?

Reply:
Oops yeah. Ok, I think it's critical that we correctly add watchers for new listings so will keep for now